cluster/addons/dashboard/dashboard-controller.yaml

```diff
@@ -56,8 +56,7 @@ spec:
           timeoutSeconds: 30
       volumes:
       - name: kubernetes-dashboard-certs
-        secret:
-          secretName: kubernetes-dashboard-certs
+        emptyDir: {}
```

Comment:
I see there was an issue opened on the dashboard discussed before, https://github.com/kubernetes/kubernetes/pull/58720/files#r166033248, but it's better to back port like this, right?

Reply:
Oh, now I see the other commit using the new dashboard. Dismiss this, please :)